### ISSUE TITLE
Reconcile invoices and credit notes

### ISSUE THREAD

**Issue by barredterra** [alyf-de/banking#118]
#111 introduced Multi-Party reconciliation. However, the following case does not seem to be covered yet:

We issue a Sales Invoice SINV-001 for $100 to Customer A.
We issue a Credit Note SINV-RET-001 for $50 to Customer B (who is a subsidiary of Customer A).
We receive a Bank Transaction for $50 from Customer A, with the description "SINV-001, SINV-RET-001".
We want to reconcile the matched Invoice and Credit Note.

**Comment by barredterra** [comment]
@marination one of our customers has now run into this case.

**Comment by marination** [comment]
@barredterra Isn't this what is required ?

![Image](https://github.com/user-attachments/assets/1fad09f3-e587-40b6-a5ac-8b58474eea02)

**Comment by barredterra** [comment]
Yes, looks good! Maybe it has been fixed meanwhile, without closing the issue?
